.travis.yml

```diff
@@ -30,8 +30,7 @@ install:
   - source activate $ENV_NAME
 
   # Download Iris 1.13 and all dependencies.
-  - conda install -c conda-forge iris=1.13
-  - conda install -c conda-forge iris-grib=0.10.1
+  - conda install -c conda-forge iris=2.0
 
   # Install our own extra dependencies (+ filelock for Iris test).
   - conda install -c conda-forge mock filelock numpy=1.13.3 pycodestyle pylint pandas python-stratify sphinx=1.7.0
```

Comment:
Maybe the numpy version needs updating? Also the Iris 1.13 comment a couple of lines up

Reply:
OK, no worries!